database/schedule/delete_test.go

```diff
@@ -5,6 +5,7 @@
 package schedule
 
 import (
+	context "context"
```

Comment:
I believe this is a typo:

```suggestion
	"context"
```